js/API/API.js

```diff
@@ -227,16 +227,11 @@ data - The data to download.
 */
 
 function downloadFile(fileName, data) {
-    var a = document.createElement("a");
-    document.body.appendChild(a);
-	
-	var blob = new Blob([data], {type: "octet/stream"}),
-	    url = window.URL.createObjectURL(blob);
-	a.href = url;
-	a.download = fileName;
-	a.click();
-	window.URL.revokeObjectURL(url);
-	a.remove();
+	var downloadlink = document.body.appendChild(document.createElement("a"));
+	downloadlink.download = fileName;
+	downloadlink.href = "data:text/plain;base64," + btoa(unescape(encodeURIComponent(data)));
```

Comment:
After some testing my conclusion is that data-uris works best in Firefox but Blobs works best in Chrome

Firefox could do about 30 megabyte file downloads with data-uris, while Chrome stopped accepting data-uris after about 1 megabyte.

I will come back with a patch that does URI-downloads for Firefox but Blob downloads for Chrome